Monika After Story/game/script-topics.rpy

```diff
@@ -3709,10 +3709,59 @@ label monika_penname:
     m 3c "They keep it hidden from everyone just so it won't affect their personal lives."
     m 3b "Pen names also help writers create something totally different from their usual style of writing."
     m 3d "It really gives the writer the protection of anonymity and gives them a lot of creative freedom."
-    if mcname.lower() != player.lower():
-        m 2c "Is '[mcname]' a pseudonym that you're using?"
-        m "You're using two different names after all."
-        m 2d "'[mcname] and [player].'"
+    m "Do you have a pen name, [player]?"
```

Comment:
since you'll be storing the pen name in a persistent var it would be better that instead of asking she just mentions the pen name the player has if he has already answered(you'll check if the persistent var is not empty) and move on with the rest of the topic, if the player hasn't answered go with the asking flow.